contracts/DepositManager.sol

```diff
@@ -10,26 +10,55 @@ import { ImmutableOwnable } from "./libs/ImmutableOwnable.sol";
 import { Rollup } from "./rollup/Rollup.sol";
 import { ITokenRegistry } from "./TokenRegistry.sol";
 
+/**
+ * @notice Interface for a contract acting as a manager of deposits for a Hubble network
+ */
 interface IDepositManager {
+    /**
+     * @notice Event when a deposit has been enqueued
+     * for eventual submission to the rollup
+     * @param pubkeyID Registered public key ID
+     * @param tokenID Registered token ID
+     * @param l2Amount Value of deposit in L2
+     * @param subtreeID Subtree this deposit will be part of (1 - n)
+     * @param depositID Deposit number in the subtree (0 - 1 << maxSubtreeDepth)
+     */
     event DepositQueued(
         uint256 pubkeyID,
         uint256 tokenID,
         uint256 l2Amount,
         uint256 subtreeID,
         uint256 depositID
     );
+    /**
+     * @notice Event when a deposit subtree is ready
+     * to be submitted to the rollup
+     * @param subtreeID Subtree ID of deposits (1 - n)
+     * @param subtreeRoot Merkilized root of subtree
```

Comment:
Nitpick: There are different variations of spellings of "Merklized", but this looks new
```suggestion
     * @param subtreeRoot Merklized root of subtree
```